app/renderer/js/main.js

```diff
@@ -267,11 +267,12 @@ class ServerManagerView {
 		}
 
 		$altIcon.textContent = realmName.charAt(0) || 'Z';
-		$altIcon.classList.add('server-icon');
+		$altIcon.classList.add('server-icons');
 		$altIcon.classList.add('alt-icon');
 
 		$parent.removeChild($img);
 		$parent.appendChild($altIcon);
+		$parent.attr('data-content', '');
```

Comment:
`attr` is not a DOM element function, correct usage here would be `setAttribute` . 
```suggestion
		$parent.setAttribute('data-content', '');
```